### PR TITLE
Multitarget Pretzel & Pretzel.Tests to net462 & netcoreapp2.2

### LINKED ISSUE #344: Multitarget Pretzel & Pretzel.Tests to net462 & netcoreapp2.2
depends #328
relates #320
prerequisite #338 

- [ ] App is working on
  - [ ] net462
  - [ ] netcoreapp2.2

- [ ] Tests are working on
  - [ ] net462
  - [ ] netcoreapp2.2

### PR DESCRIPTION
**Pull request by biohazard999**
fixes #344

- [x] App is working on
  - [x] net462
  - [x] netcoreapp2.2

- [x] Tests are working on
  - [x] net462
  - [x] netcoreapp2.2
  - [x] Appveyor for both platforms

Had to make some changes to the way [open cover works (or not)](https://github.com/OpenCover/opencover/issues/789)

@laedit Should be ready to review & test

### CODE REVIEW

File: src/Pretzel.Logic/Properties/Resources.resx [hunk 1]
```diff
@@ -118,18 +118,6 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
-  <data name="faviconico" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\Resources\favicon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```
Comment (laedit) [inline src/Pretzel.Logic/Properties/Resources.resx]: Images aren't supported in resources?

File: src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs [hunk 2]
```diff
@@ -992,9 +992,17 @@ public void page_with_false_date_is_not_processed()
             var siteContext = generator.BuildContext(@"C:\TestSite", @"C:\TestSite\_site", false);
 
             Assert.Empty(siteContext.Pages);
+
+            var x = trace.ToString();
```
Comment (laedit) [inline src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs]: Why this first execution?

File: src/Pretzel.Logic/Properties/Resources.resx (hunk 1 above)
Comment (laedit) [reply src/Pretzel.Logic/Properties/Resources.resx]: Ok I have seen the link to the issue in corefx.

File: src/Pretzel.Logic/Pretzel.Logic.csproj [hunk 3]
```diff
@@ -73,27 +72,11 @@
...
-    <None Include="Resources\RazorCsProject\Paginator_cs" />
-    <None Include="Resources\RazorCsProject\SiteContext_cs" />
-    <None Include="Resources\RazorCsProject\Tag_cs" />
-    <None Include="Resources\RazorCsProject\AssemblyInfo_cs" />
     <Compile Update="Properties\RazorWiki.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>RazorWiki.resx</DependentUpon>
     </Compile>
-    <Compile Update="Properties\Resources.Designer.cs">
```
[the remark this answers is not in the document]
Comment (biohazard999) [reply src/Pretzel.Logic/Pretzel.Logic.csproj]: It is supported, somehow my studio decided to f*ck up the file nesting. Fixed.

File: src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs (hunk 2 above)
Comment (biohazard999) [reply src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs]: Debug code slipped in. Removed.

File: src/Pretzel.Logic/Pretzel.Logic.csproj (hunk 3 above)
[the remark this answers is not in the document]
Comment (laedit) [reply src/Pretzel.Logic/Pretzel.Logic.csproj]: `Resources.Designer.cs` is still deleted in file changes, is that normal?

[the remark this answers is not in the document]
Comment (biohazard999) [reply src/Pretzel.Logic/Pretzel.Logic.csproj]: Did you look at all the changes?

[the remark this answers is not in the document]
Comment (laedit) [reply src/Pretzel.Logic/Pretzel.Logic.csproj]: Sorry, I was talking about the file `Resources.Designer.cs` which I thought was deleted, but it is just Github which doesn't load diff for generated files.

**Comment by biohazard999** [comment]
Just a quick look on Ubuntu for motivation:

VirtualBox_Ubuntu 18_24_10_2019_14_35_32

**Comment by biohazard999** [comment]
I am totally impressed:

VirtualBox_Ubuntu 18_24_10_2019_14_40_41

🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

**Comment by laedit** [comment]
Awesome! :D

**Pull request merged**